### PR TITLE
Code Review Groups: add group management UI to modal

### PR DESCRIPTION
**Pull request by bencodeorg**
Puts rough version of Code Review Groups manager in a modal (hidden behind an experiment flag) where it will ultimately appear. We should continue development of this feature (hooking up to back end, addtional UI work) here rather than in Storybook.

image

## Testing story

Tested manually.

### CODE REVIEW

File: apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx [hunk 1]
```diff
@@ -1,32 +1,80 @@
...
+        color={Button.ButtonColor.gray}
+        text={i18n.manageCodeReviewGroups()}
+        icon="comment"
+      />
+      <StylizedBaseDialog
+        title={i18n.codeReviewGroups()}
+        body={codeReviewGroupsManager}
+        isOpen={isDialogOpen}
+        handleClose={onDialogClose}
+        fixedWidth={1000}
```
Comment (molly-moen) [inline apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx]: nit: constant for 1000?

File: apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx [hunk 2]
```diff
@@ -1,32 +1,80 @@
...
+      />
+    </div>
+  );
 }
 
+ManageCodeReviewGroups.propTypes = {buttonContainerStyle: PropTypes.object};
+
+const names = [
```
Comment (molly-moen) [inline apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx:42]: nit: add a todo to remove this fake data at some point

**Review by molly-moen — APPROVED** [review]
LGTM, just a couple nits

File: apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx:42]: Added a Jira link (and included this in the cleanup task).

**Pull request merged**